libs/clients/national-registry/v3/src/lib/utils.ts

```diff
@@ -0,0 +1 @@
+export type ExcludesFalse = <T>(x: T | null | undefined | '') => x is T
```

Comment:
refactor this away

Reply:
100%, i'll do it in a separate refactor PR